package.json

```diff
@@ -46,7 +46,7 @@
     "jest-coverage": "yarn run jest -- --coverage",
     "lint": "eslint . --cache && yarn run lint-docs",
     "lint-docs": "eslint --config ./.eslintrc-docs.json --no-ignore --ext md ./docs/*.md",
-    "prettier": "prettier --write 'packages/*/src/**.js' --single-quote --trailing-comma=all --bracket-spacing=false",
+    "prettier": "prettier --write 'packages/*/src/**/*.js' --single-quote --trailing-comma=all --bracket-spacing=false",
```

Comment:
Turns out we didn't run prettier on everything. Oops, now we have a lot more reformatting to go! cc @SimenB.